payment-cards/pci-compliance/saqs.md

```diff
@@ -4,24 +4,34 @@ title: Self Assessment Questionnaires
 permalink: /payment-cards/pci-compliance/saqs/
 ---
 
-To be [PCI Compliant](/payment-cards/pci-compliance/) as a merchant, you'll need to fill out an SAQ, or Self Assessment Questionnaire.
+As a merchant, the least burdensome way to become [PCI Compliant](/payment-cards/pci-compliance/) is to fill out a Self Assessment Questionnaire (the next step up would be to hire a [QSA](/glossary/#qsa)).
 
 Most likely, the 2 SAQ's that are relevant to you if you accept payments online are [SAQ A](#saq-a) and [SAQ A-EP](#saq-a-ep). For a full list, see [Understanding the SAQs for PCI DSS version 3](https://www.pcisecuritystandards.org/documents/Understanding_SAQs_PCI_DSS_v3.pdf).
 
-Following SAQ rules is not enough to ensure a secure payments flow. Keeping up with industry-standards is important to make sure payment information stays secure.
+<div class="popout warning">
+Following SAQ rules is not enough to ensure a secure payments flow. Keeping up with industry standards is important to make sure payment information stays secure.
+</div>
 
 ## SAQ A
 
-The easiest. Requires use of an iframe or a redirect so that payment information is entered directly on the [provider's](/glossary/#provider) domain.
+This level is designed to be easy to integrate and to expose the merchant to as little fraud risk as possible. SAQ A requires use of an iframe or a redirect so that payment information is entered directly on a [PCI Level 1](/payment-cards/pci-compliance/levels/) compliant domain, like a [payment gateway](/glossary/#gateway).
 
-> Card-not-present merchants (e-commerce or mail/telephone-order) that have fully outsourced all cardholder data functions to PCI DSS validated third-party service providers, with no electronic storage, processing, or transmission of any cardholder data on the merchant’s systems or premises.
+Integrations using JavaScript to tokenize card information that were previously SAQ A under PCI v2 are now SAQ A-EP. Check out the [integrations page](/payment-cards/pci-compliance/integrations/) for more information.
+
+iOS and Android payment integrations need no modification from PCI v2 to v3. They still qualify for SAQ A without needing to employ iframes or redirects.
```

Comment:
iOS and Android /native/ integrations (a webview to a not compliant site) wouldn't necessarily get them around it.